docs/src/main/sphinx/release/release-386.md

```diff
@@ -0,0 +1,61 @@
+# Release 386 (15 June 2022)
+
+## General
+
+* Improve fault-tolerant execution performance by updating default configuration
+  values. ({issue}`12646`)
+* Improve exchange spooling file enumeration speed. ({issue}`12615`)
+* Remove support for Phoenix 4. ({issue}`12772`)
+
+## JDBC driver
+
+* Add configuration property `assumeLiteralUnderscoreInMetadataCallsForNonConformingClients`
+  for situations where applications do not properly escape schema or table names
+  in calls to `DatabaseMetaData`. ({issue}`12672`)
+
+## Accumulo connector
+
+* Disallow creating a view in a non-existent schema. ({issue}`12475`)
+
+## Delta Lake connector
+
+* Improve query performance and scheduling on tables with many small files. ({issue}`12755`)
+* Disallow reading tables if `delta.columnMapping.mode` table property is
+  specified. ({issue}`12621`)
+* Add target maximum file size with 1 GB default. The target max file
+  size can be set with the `target_max_file_size` session property or the
+  `target-max-file-size` configuration property. ({issue}`12820`)
+
+## Hive connector
+
+* Fix Parquet incompatibility with Apache Hive when writing low-precision
```

Comment:
"low-precision" doesn't mean anything. Let's be more specific and say "with precision smaller than X".

Also, I'd rephrase as:

> Fix incompatibility with Apache Hive when writing decimal values with precision smaller than X with the experimental Parquet writer.